specutils/processing/__init__.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import absolute_import
+
+from .smoothing import *  # noqa
```

Comment:
I renamed the `processing` directory to be `smoothing`, so with the import in the `__init__` it will functionally look like `from specutils.smoothing import X`.